debian/control

```diff
@@ -10,8 +10,8 @@ Build-Depends: debhelper (>= 11),
 Standards-Version: 4.3.0
 Homepage: https://github.com/intel/metrics-library
 
-Package: intel-metrics-library
-Replaces: libigdml1
+Package: libigdml1
+Provides: intel-metrics-library
```

Comment:
@mckees  The reason to use `Provides` here is to not break existing Intel documentation and Dependencies for Intel internal stack (a package at Intel depends on `intel-metrics-library`) ? Because other than that, i do not see the reason to do this, this package is something new in Ubuntu and no package in Ubuntu depends on `intel-metrics-library`. If that is only for Intel, i don't know if we want to do this

Reply:
I'm using it here to satisfy the desire to have a more friendly name while keeping the package name the same as the soname

Reply:
@mckees  I don't think `Provides` is used to have pretty name of the package, there must be another reason to justify it:
- satisfy an existing dependency with a different name (pretty or not)